v2/emailpassword/common-customizations/sessions/update-session-data.mdx

```diff
@@ -57,7 +57,7 @@ let sessionHandles = Session.getAllSessionHandlesForUser(userId);
 //highlight-start
 // we update all the session's data for this user
 sessionHandles.forEach(async (handle) => {
-      let currSessionData = await Session.getSessionData(handle);
+      let currSessionData = (await Session.getSessionInformation(handle)).userDataInDatabase;
```

Comment:
I don't like `.userDataInDatabase`. Maybe it should be `.sessionData`

Reply:
It does make sense but this will require a change in the node SDK and core (if we want to keep the naming consistent across). We could just make the change in the node SDK but that might introduce confusion

Reply:
You can make the change just to the node SDK.

Reply:
Done